.travis.yml

```diff
@@ -129,6 +131,7 @@ matrix:
 install:
   - source bin/install_travis.sh
 script:
+  - bin/travis_clang.sh
```

Comment:
Set an environment variable (eg: TEST_CLANG_FORMAT) to true in one of the jobs and check that the variable is true before calling this script.